### PR TITLE
Make full identifier type default

### PR DESCRIPTION
**Pull request by ddfreyne**

### CODE REVIEW

File: lib/nanoc/cli/commands/create-site.rb [hunk 1]
```diff
@@ -23,7 +23,7 @@ def array_to_yaml(array)
 # The syntax to use for patterns in the Rules file. Can be either `"glob"`
 # (default) or `null`. The former will enable glob patterns, which behave like
```
Comment (bobthecow) [inline lib/nanoc/cli/commands/create-site.rb]: ^ update the comment here?

Comment (ddfreyne) [inline lib/nanoc/cli/commands/create-site.rb]: Done

**Comment by ddfreyne** [comment]
Rebased onto master, ready to review.

**Comment by ddfreyne** [comment]
Build fails because of some JRuby oddness. I restarted the build.

**Comment by ddfreyne** [comment]
Well, I am not sure there is much I can do about `#fnmatch` crashing with a Java `ArrayIndexOutOfBoundsException` (except report it).

**Comment by ddfreyne** [comment]
Can’t reproduce the issue locally. I marked JRuby 9000 as an allowed failure, because it failing doesn’t necessary imply a bug in nanoc.

**Comment by ddfreyne** [comment]
Post-merge +1s are OK :)

**Pull request merged**